src/pages/civic/_from.vue

```diff
@@ -68,5 +68,42 @@ export default {
       });
     },
   },
+  head() {
+    let title, description;
+    if (this.creator) {
+      const name = this.creator.displayName.trim();
+      title = this.$t('CivicEntryPage.Og.Title', { name });
+      description = this.creator.creatorPitch;
+    } else {
+      title = this.$t('CivicPage.Og.Title');
+      description = this.$t('CivicPage.Og.Description');
+    }
+    return {
+      title,
+      meta: [
+        {
+          hid: 'og:title',
+          property: 'og:title',
+          content: title,
+        },
+        {
+          hid: 'description',
+          name: 'description',
+          content: description,
+        },
+        {
+          hid: 'og:description',
+          property: 'og:description',
+          content: description,
+        },
+        {
+          hid: 'og:image',
+          property: 'og:image',
+          content: 'https://liker.land/images/og/civic-v2.png',
+        },
+      ],
+      link: [{ rel: 'canonical', href: `${this.$route.path}` }],
```

Comment:
canonical might not be path since we have querystring `from`, please remove this line for now since it is irrelevant to this PR